p4_pdpi/BUILD.bazel

```diff
@@ -288,3 +288,20 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_library(
+    name = "p4_runtime_session_extras",
+    srcs = ["p4_runtime_session_extras.cc"],
+    hdrs = ["p4_runtime_session_extras.h"],
+    deps = [
+        ":ir",
+        ":ir_cc_proto",
+        ":p4_runtime_session",
+        ":pd",
+        "//gutil:proto",
+        "@com_github_p4lang_p4runtime//:p4runtime_cc_proto",
+        "@com_google_absl//absl/status",
+        "@com_google_absl//absl/strings",
+        "@com_google_protobuf//:protobuf",
+    ],
+)
```

Comment:
@bibhuprasad-hcl p4_runtime_session_extras_test addition to BUILD.bazel is missing